PurchasesTests/BackendTests.swift

```diff
@@ -92,12 +92,22 @@ class BackendTests: XCTestCase {
         let isRestore = arc4random_uniform(2) == 0
         let observerMode = arc4random_uniform(2) == 0
 
-        backend?.postReceiptData(receiptData, appUserID: userID, isRestore: isRestore, productIdentifier: nil,
-                price: nil, paymentMode: RCPaymentMode.none, introductoryPrice: nil, currencyCode: nil,
-                subscriptionGroup: nil, discounts: nil, presentedOfferingIdentifier: nil, observerMode: observerMode,
-                completion: { (purchaserInfo, error) in
-                    completionCalled = true
-                })
+        backend?.postReceiptData(receiptData,
```

Comment:
these are mostly reformatting + adding the subscriber attributes param

Reply:
tests specifically for subscriber attributes in `RCBackend` are in `BackendSubscriberAttributesTests`